### PR TITLE
ops: Add oneshot deployment container

### PR DESCRIPTION
**Pull request by mslipper**
Adds a one-shot container that runs both op-geth and op-node in the same container. It also embeds the configuration for a specific network in the container in order to cut down on deployment errors.

See README.md for full usage and architecture.

### CODE REVIEW

File: ops-bedrock/oneshot/op-init.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	echo -n "pwd" > "$GETH_DATA_DIR"/password
+	echo -n "$BLOCK_SIGNER_PRIVATE_KEY" | sed 's/0x//' > "$GETH_DATA_DIR"/block-signer-key
+	geth account import \
+		--datadir="$GETH_DATA_DIR" \
+		--password="$GETH_DATA_DIR"/password \
+		"$GETH_DATA_DIR"/block-signer-key
+else
+	echo "$GETH_KEYSTORE_DIR exists."
+fi
+
```
Comment (protolambda) [inline ops-bedrock/oneshot/op-init.sh:43]: should remove this when the --mine and --miner.etherbase flags are removed

File: ops-bedrock/oneshot/s6-rc.d/op-geth/run [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	--ws.api=debug,eth,txpool,net,engine \
+	--syncmode=full \
+	--nodiscover \
+	--maxpeers="$OP_GETH_MAX_PEERS" \
+	--networkid=$CHAIN_ID \
+	--unlock=$BLOCK_SIGNER_ADDRESS \
+	--mine \
+	--miner.etherbase=$BLOCK_SIGNER_ADDRESS \
+	--password="$GETH_DATA_DIR"/password \
+	--allow-insecure-unlock \
```
Comment (protolambda) [inline ops-bedrock/oneshot/s6-rc.d/op-geth/run:35]: remove these

File: ops-bedrock/oneshot/op-init.sh [hunk 3]
```diff
@@ -0,0 +1,40 @@
+#!/command/with-contenv bash
+
+set -eu
+
+GETH_DATA_DIR=/db
+GETH_CHAINDATA_DIR="$GETH_DATA_DIR/geth/chaindata"
+GETH_KEYSTORE_DIR="$GETH_DATA_DIR/keystore"
+GENESIS_FILE_PATH="/etc/op-geth/genesis.json"
+BLOCK_SIGNER_PRIVATE_KEY="3e4bde571b86929bf08e2aaad9a6a1882664cd5e65b96fff7d03e1c4e6dfa15c"
```
Comment (tynes) [inline ops-bedrock/oneshot/op-init.sh:9]: This env var looks unused

Comment (mslipper) [reply ops-bedrock/oneshot/op-init.sh]: good catch, fixed

**Review by tynes — APPROVED** [review]

**Pull request merged**